jsk_panda_robot/jsk_panda_teleop/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(jsk_panda_teleop)
+
+
+find_package(catkin REQUIRED
+  message_generation
+  )
+
+add_service_files(
+  FILES
+  ControlBilateral.srv
+  )
+
+generate_messages()
+
+catkin_package()
+
+install(DIRECTORY config launch python script
```

Comment:
use `catkin_install_python` https://github.com/jsk-ros-pkg/jsk_recognition/pull/2743 ? (you do not have to check on 20.04.)

Reply:
Maybe fixed